build_tools/docker/gradle-android/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# An image for cross-compiling IREE's TFLite Java Bindings with Gradle and
+# CMake.
+
+FROM gcr.io/iree-oss/cmake@sha256:9d9953acf5ca0cf1ff3e8de32f10f24dfab1c4e8ec5d1fc047f556024ee4bed6
+
+# Tools for downloading/installing packages.
+RUN apt-get update \
+  && apt-get install -y \
+  unzip \
+  wget
+
+# Download and install openjdk-11
+ARG JDK_VERSION=11
+WORKDIR /install-jdk
+RUN apt-get update && apt-get install -y openjdk-11-jdk
+
+# Download and install Gradle
+ARG GRADLE_VERSION=7.1.1
+ARG GRADLE_DIST=bin
+WORKDIR /install-gradle
+RUN cd /opt && \
+    wget -q https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-${GRADLE_DIST}.zip && \
+    unzip gradle*.zip && \
+    rm gradle*.zip && \
+    ln -s /opt/gradle-${GRADLE_VERSION}/bin/gradle /usr/bin/gradle
+
+# Download and install Android SDK
+ARG ANDROID_SDK_VERSION=7583922
```

Comment:
Can you document where this version number came from / how to find newer versions? https://source.android.com/setup/start/build-numbers doesn't have this particular number listed